app/views/episode_media/media/_new.html.erb

```diff
@@ -55,15 +55,17 @@
   </div>
 
   <%# step 3a: hidden original_url field pointing to the uploaded temporary s3 file %>
-  <div class="form-floating d-none input-group" data-upload-target="success">
+  <div class="form-floating input-group <%= "d-none" unless upload_uploaded?(media) %>" data-upload-target="success">
```

Comment:
Had to change a couple `d-none`s rendered by the server.  If we already have an uploaded file, skip straight to this step 3 which shows the uploaded file name, with the is-changed orange border.